docs/reference/runtime-fields.asciidoc

```diff
@@ -1,15 +1,17 @@
-[[alerts-runtime-fields]]
-== Add runtime fields to detection alerts
+[[runtime-fields]]
+== Create runtime fields in {elastic-sec}
```

Comment:
what is `{elastic-sec}`?

Reply:
@stephmilovic It's a variable (called an "attribute" in Asciidoc) used for the name "Elastic Security." There are a few other attributes; they're always in curly brackets like this.